schemas/com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "self": {
+                "vendor": "com.snowplowanalytics.datamodeling",
+                "name": "config",
+                "version": "1-0-0",
+                "format": "jsonschema"
+        },
+        "type": "object",
+        "properties": {
+                "schema": {
+                        "type": "string",
+                        "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$",
+                        "maxLength": 256
+                },
+                "data": {
+                        "type": "object",
+                        "properties": {
+                                "enabled": {
+                                        "type": "boolean"
+                                },
+                                "lockType": {
+                                        "type": "string",
+                                        "enum": [
+                                                "hard",
+                                                "soft"
+                                        ]
+                                },
+                                "storage": {
+                                        "type": "string",
+                                        "enum": [
+                                                "Default",
+                                                "BigQuery"
+                                        ]
+                                },
+                                "playbooks": {
+                                        "type": "array",
+                                        "items": {
+                                                "type": "object",
+                                                "properties": {
+                                                        "playbook": {
+                                                                "type": "string",
+                                                                "minLength": 1,
+                                                                "maxLength": 64
+                                                        },
+                                                        "dependsOn": {
+                                                                "type": "array",
+                                                                "items": {
+                                                                        "type": "string",
+                                                                        "minLength": 1,
+                                                                        "maxLength": 64
+                                                                },
+                                                                "minItems": 0,
+                                                                "maxItems": 64
```

Comment:
Same for maxItems in arrays - no need to impose the restriction in the schema if it doesn't exist in practice.